src/test/ui/consts/const-pattern-irrefutable.rs

```diff
@@ -19,8 +19,8 @@ use foo::d;
 const a: u8 = 2;
 
 fn main() {
-    let a = 4; //~ ERROR refutable pattern in local binding: `_` not covered
-    let c = 4; //~ ERROR refutable pattern in local binding: `_` not covered
-    let d = 4; //~ ERROR refutable pattern in local binding: `_` not covered
+    let a = 4; //~ ERROR refutable pattern in local binding: `0u8..=1u8` not covered
```

Comment:
The error message doesn't seem to cover all the missing patterns? e.g. `3` is not covered?

Don't have to fix that in this PR tho; can always do a follow up...